src/containerbuddy/config.go

```diff
@@ -80,6 +85,41 @@ const (
 	defaultStopTimeout int = 5
 )
 
+func parseInterfaces(raw json.RawMessage) ([]string, error) {
+	if raw == nil {
+		return nil, nil
```

Comment:
Do we want to return `nil, nil` here or should we be returning an empty array? I feel like the latter is going to be less likely to result in a null-pointer exception down the road.